accelerator/errors.h

```diff
@@ -47,16 +47,15 @@ extern "C" {
 #define SC_ERROR_MASK 0x07
 /** @} */
 
-/** @name http error code */
-/** @{ */
-#define SC_BAD_REQUEST 400
-#define SC_NOT_FOUND 404
-/** @} */
-
 /* status code */
 typedef enum {
   SC_OK = 0, /**< No error */
 
+  SC_HTTP_OK = 200,        /**< HTTP response OK */
+  SC_BAD_REQUEST = 400,    /**< HTTP response, error when parsing request */
```

Comment:
I expect all HTTP response codes have `SC_HTTP_` prefix for consistency.

Reply:
Reference: https://github.com/phusion/nginx/blob/master/src/http/ngx_http_request.h